README.md

```diff
@@ -1,9 +1,55 @@
 # Gruntwork Installer
 
-At [Gruntwork](http://www.gruntwork.io/), we've developed a number of scripts and binaries, most of them in private
+`gruntwork-install` is a bash script you run to easily download and install "Script Modules" written by Gruntwork. 
+
+### Script Modules
+
+A Script Module is a package of one or more bash scripts and/or binaries maintained by Gruntwork that are meant to be 
+reusable in many different contexts. For example, we have Script Modules for installing a CloudWatch Logs agent, optimizing
+syslog settings, setting up automatic security updates, and more. 
+
+Our Script Modules are contained in the `modules/` folder of different GitHub repos since they often come with the 
+different [Infrastructure Packages](https://blog.gruntwork.io/gruntwork-infrastructure-packages-7434dc77d0b1#.6bwor6wxc) 
+we sell.
```

Comment:
I'm glad we're having this discussion since our "ontology" is an important part of keeping things manageable as we add more functionality (reminder: this convo is public).

So, I really like the term "module" because its official definition is just so spot on:

> each of a set of standardized parts or independent units that can be used to construct a more complex structure, such as an item of furniture or a building.

"App" to me does not imply "composable."

To your point, though, our modifier "Script" now seems outdated. Clearly, downloading a binary isn't any kind of "Script" module. But I think it's still a module (per the definition above), so I propose that we introduce different module types:

- "Script Module" (for `install.sh` scripts)
- "Binary Module" (for downloading a release directly from a repo; BTW, do we have any distinction between a repo like fetch where releases are made available and "Binary Modules" where we intend for such releases to be used as Modules?

Anyway, I could also imagine "Docker Modules" or other types.

Let me know your thoughts on this, because I'd like to wrap this up ASAP and incorporate into the Vault AMI updates I'm doing right now.